### PR TITLE
Build dfe2736

### PR DESCRIPTION
**Pull request by mareklibra**
References docker.io/mareklibra/kubevirt-web-ui:dfe2736842f02b6f63880436eb8e9961a86e5d39

This is not long-term approach for releases.
Proper deployment is WIP recently, this PR should ease verification in the meantime.

### CODE REVIEW

**Comment by coreos-ui** [comment]
Can one of the admins verify this patch?

**Comment by coreos-ui** [comment]
Can one of the admins verify this patch?

**Comment by mareklibra** [comment]
I'm sorry for confusion, opened mistakenly.

File: kubevirt/kubevirt-web-ui.yaml [hunk 1]
```diff
@@ -0,0 +1,66 @@
+apiVersion: v1
+kind: ReplicationController
```
Comment (alecmerdler) [inline kubevirt/kubevirt-web-ui.yaml]: Why not a `Deployment`?

**Pull request closed without merging**